src/components/ChallengeEditor/index.js

```diff
@@ -355,6 +355,7 @@ class ChallengeEditor extends Component {
           newChallenge[field][index][option.key] = option.name
         }
       }
+      console.log(newChallenge)
```

Comment:
@dashu-baba let's remove all the debugging console logs.

Reply:
removed